examples/adrv9002-iiostream.c

```diff
@@ -18,13 +18,11 @@
 #include <errno.h>
 #include <stdbool.h>
 
-#define ARGS(fmt, ...)	__VA_ARGS__
-#define FMT(fmt, ...)	fmt
-#define error(...) \
-	printf("%s, %d: ERROR: " FMT(__VA_ARGS__, 0)"%s", __func__, __LINE__, ARGS(__VA_ARGS__, ""))
+#define error(fmt, ...) \
+	printf("%s, %d: ERROR: " fmt, __func__, __LINE__, ##__VA_ARGS__)
 
-#define info(...) \
-	printf("%s, %d: INFO: " FMT(__VA_ARGS__, 0)"%s", __func__, __LINE__, ARGS(__VA_ARGS__, ""))
+#define info(fmt, ...) \
+	printf("%s, %d: INFO: " fmt, __func__, __LINE__, ##__VA_ARGS__)
```

Comment:
Be careful with these changes... Did you tested it on some older windows compilers? IIRC, I had to do all of that gymnastics with the macros exactly because of windows.

AFAIK, ##__VA_ARGS__ is a gnu extension and I thought we were not using it? So I'm a bit confused how the above compiles on window. With what compiler did you tested it? Maybe newer compilers support it...

Reply:
Is there a reason for why we need the examples to be built without GNU extensions?

Reply:
Note that I'm aware. I guess it's just a matter of portability since the examples should run in multiple platforms.

> The ##_VA_ARGS is a GNU extension indeed but has been standardized in C2x.

Yes, I assumed you're compiling using a recent standard.

> I wouldn't worry about this not working with older windows compilers, as long as it's not part of the library itself.

I'm not going to argue much here as I'm not aware about what is our policy (commitment) in terms of compilers (and C standards) support. But this always opens the door to get someone opening issues or complaining a given piece of code does not work in the environment.